### DISCUSSION TITLE
Idea

### DISCUSSION THREAD

**Discussion by astrowke** [webstudio-is/webstudio-community#67] [Ideas]
Why don't you add a redo and undo buttons? Really gonna help with the editing!

**Comment by kof** [comment]
We think undo/redo is basic, everyone knows the shortcut. Figma doesn't have the buttons either.

**Comment by itsNintu** [comment]
@astrowke as in, we already have undo/redo built in with the standardised keyboard shortcuts (cntrl/cmd + C/V). 

I agree with Oleg on this one and don't think they need to take up valuable UI space. 

With the build muscle memory, users will first try the keyboard shortcut and then try to find a button on the UI if the shortcut doesn't work. 

That said appreciate the suggestion! Keep em coming ♡
